### PR TITLE
vivek/Mar-28

### PR DESCRIPTION
**Pull request by vivek1214-git**
post-graduation select dropdown validation

### CODE REVIEW

File: resources/views/components/duolingo-form.blade.php [hunk 1]
```diff
@@ -62,10 +62,15 @@ class="form-control @error('duolingo_overall_score') is-invalid @enderror" name=
             160
         </label>
     </div>
-
+    @error('duolingo_scale')
+    <div class="invalid-feedback">
+        {{ $message }}
+    </div>
+@enderror
+</div>
```
Comment (ithqindia) [inline resources/views/components/duolingo-form.blade.php:70]: code not formatted

File: resources/views/components/duolingo-form.blade.php [hunk 2]
```diff
@@ -104,6 +115,72 @@ class="form-control @error('duolingo_overall_score') is-invalid @enderror" name=
...
     </div>
     <div class="my-5">
-        <button type="submit" class="btn btn-primary">Submit</button>
+        <button type="submit" class="btn btn-primary" id='submit-btn'>Submit</button>
     </div>
 </div>
+
+@section('scripts')
+    <script>
+$("#duolingo_exam_date").daterangepicker({
```
Comment (ithqindia) [inline resources/views/components/duolingo-form.blade.php:124]: code not formatted

File: resources/views/components/duolingo-form.blade.php [hunk 3]
```diff
@@ -104,6 +115,72 @@ class="form-control @error('duolingo_overall_score') is-invalid @enderror" name=
...
+                    validators: {
+                            notEmpty: {
+                                message: 'Exam date is required',
+                            },
+                        },
+                    },
+                    duolingo_user_id: {
+                        validators: {
+                            notEmpty: {
+                                message: 'UserId is required',
```
Comment (ithqindia) [inline resources/views/components/duolingo-form.blade.php:144]: Duolingo user id is required

File: resources/views/components/gre-general-form.blade.php [hunk 4]
```diff
@@ -44,20 +44,30 @@ class="form-control @error('gre_general_test_registration_number') is-invalid @e
...
         <div class="col-4">
             <label for="TextInput" class="form-label">Verbal<span style="color:red; font-size:20px">*</span></label>
-            <input type="text" id="verbal" class="form-control" name="gre_general_test_verbal"
+            <input type="text" id="verbal" class="form-control  @error('gre_general_test_verbal') is-invalid @enderror" name="gre_general_test_verbal"
                 value="{{ old('gre_general_test_verbal', $extraScoreGreGeneral->gre_general_test_verbal ?? '') }}">
+                @error('gre_general_test_verbal')
+                <div class="invalid-feedback">
+                    {{ $message }}
+                </div>
+            @enderror
```
Comment (ithqindia) [inline resources/views/components/gre-general-form.blade.php:53]: code not formatted

File: resources/views/components/gre-general-form.blade.php [hunk 5]
```diff
@@ -44,20 +44,30 @@ class="form-control @error('gre_general_test_registration_number') is-invalid @e
...
             <label for="TextInput" class="form-label">Quantitative<span
                     style="color:red; font-size:20px">*</span></label>
-            <input type="text" id="quantitative" class="form-control" name="gre_general_test_quantitative"
+            <input type="text" id="quantitative" class="form-control @error('gre_general_test_quantitative') is-invalid @enderror" name="gre_general_test_quantitative"
                 value="{{ old('gre_general_test_quantitative', $extraScoreGreGeneral->gre_general_test_quantitative ?? '') }}">
+                @error('gre_general_test_quantitative')
+                <div class="invalid-feedback">
+                    {{ $message }}
+                </div>
+            @enderror
```
Comment (ithqindia) [inline resources/views/components/gre-general-form.blade.php:64]: code not formatted

File: resources/views/components/gre-general-form.blade.php [hunk 6]
```diff
@@ -88,11 +98,16 @@ class="form-control @error('gre_general_test_registration_number') is-invalid @e
                     <option value="6">6</option>
                 @endif
             </select>
+            @error('gre_general_test_analitical_writing')
+            <div class="invalid-feedback">
+                {{ $message }}
+            </div>
+        @enderror
```
Comment (ithqindia) [inline resources/views/components/gre-general-form.blade.php:105]: code not formatted

File: resources/views/components/gre-general-form.blade.php [hunk 7]
```diff
@@ -88,11 +98,16 @@ class="form-control @error('gre_general_test_registration_number') is-invalid @e
...
         </div>
     </div>
     <div class="row">
         <div class="col-6 mt-3">
-            <p>Are you planning to take the test again</p>
+            <p>Are you planning to take the test again<span style="color:red; font-size:20px">*</span></p>
```
Comment (ithqindia) [inline resources/views/components/gre-general-form.blade.php:110]: Is this field required?

**Review by ithqindia — CHANGES_REQUESTED** [review]
Are you planning to take the test again - I think this is not mandatory and should be set to no by default

File: resources/views/components/gre-general-form.blade.php (hunk 7 above)
Comment (vivek1214-git) [reply resources/views/components/gre-general-form.blade.php:110]: sir I was checked the backend validation so their was required field for this test again field so I was applied on frontend also

**Review by ithqindia — CHANGES_REQUESTED** [review]
Please check formatting

File: resources/views/me/post-graduation-form.blade.php [hunk 8]
```diff
@@ -4,7 +4,7 @@
 @section('main-content')
     @include('me.info-panel')
     <div class="card mb-a8 p-10">
-        <form class="row g-3" method="post" action="/me/post-graduation" enctype="multipart/form-data">
+        <form class="row g-3" method="post" action="/me/hsc" enctype="multipart/form-data" id='post-graduation-form'>
```
Comment (ithqindia) [inline resources/views/me/post-graduation-form.blade.php:7]: ❓

**Comment by ithqindia** [comment]
Merged locally

**Pull request closed without merging**